e2e/apprunner/apprunner_test.go

```diff
@@ -165,6 +165,18 @@ var _ = Describe("App Runner", Ordered, func() {
 			}
 		})
 
+		It("should return correct secrets", func() {
+			fmt.Printf("\n\nsecrets: %+v\n\n", svc.Secrets)
+			expectedSecrets := map[string]string{
+				"COPILOT_ENVIRONMENT_NAME": envName,
+				"COPILOT_SERVICE_NAME":     feSvcName,
```

Comment:
I might have missed something from one of the previous PRs - why are `COPILOT_ENVIRONMENT_NAME` and `COPILOT_SERVICE_NAME`  also in the `Secrets` section of `copilot svc show`?

Reply:
Will remove this. They are redundant fields.